int16 state
int16 setpoint
int16 steering
int16 throttle
